### ISSUE TITLE
Show the number of reverse dependencies next to the link

### ISSUE THREAD

**Issue by carols10cents** [rust-lang/crates.io#496]
If someone wants to use the number of reverse dependencies as a way to evaluate a crate, it would save a click to show the number of reverse dependencies next to the link to the reverse dependencies list. It could also help the person decide whether or not the link would be worth clicking.

**Comment by autodidaddict** [comment]
Is there a script or something we can use to fill a sample set of dummy crates locally so that we can test things like this? There are some restrictions on what we can publish locally with `cargo` so I thought it'd be easier if we could just load up a database of fake crates.

**Comment by carols10cents** [comment]
@autodidaddict not really. How comfortable are you with SQL? If you have two crates published, it shouldn't be too bad to insert an association record for them in the database directly.
